pages/index.tsx

```diff
@@ -18,6 +18,9 @@ const IndexPage: React.SFC<{
     if (!token) {
         return <Error statusCode={404} />;
     }
+    if (role === "candidate" && candidateName !== undefined) {
+        candidateName = undefined;
+    }
```

Comment:
Before that, when a candidate was logged and got candidate_id in query, it tried to create a channel automatically, better to get an error, or just remove candidateName (who is a string who contain the canme of candidate we want to chat) like i do it here?